src/Search/Search.js

```diff
@@ -1,20 +1,23 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import TextField from '@material-ui/core/TextField';
 import CardList from '../CardsInfo.json';
-import '../Styles/Search.scss';
+import CardsContext from '../Contexts/CardsContext';
+import './Search.scss';
+
+const Search = () => {
+  // eslint-disable-next-line
+  const [cards, setCards] = useContext(CardsContext);
 
-const Search = ({ setItems }) => {
   const handleChange = (e) => {
-    const str = e.currentTarget.value;
-    const items = CardList.cards.filter((item) => item.name.toLowerCase().includes(str.toLowerCase()));
-    setItems(items);
+    const inputText = e.currentTarget.value;
+    setCards(CardList.cards.filter((card) => card.name.toLowerCase().includes(inputText.toLowerCase())));
   };
 
   return (
     <div className="searchInput">
       <TextField
```

Comment:
It's still purple. MaterialUI is not needed in this project AT ALL. Write your own input.